lib/services/tx_queue_client/default_tx_queue_client.dart

```diff
@@ -11,17 +11,17 @@ import 'package:provenance_wallet/services/account_service/account_service.dart'
 import 'package:provenance_wallet/services/account_service/model/account_gas_estimate.dart';
 import 'package:provenance_wallet/services/account_service/transaction_handler.dart';
 import 'package:provenance_wallet/services/models/account.dart';
-import 'package:provenance_wallet/services/tx_queue_service/models/sembast_gas_estimate.dart';
-import 'package:provenance_wallet/services/tx_queue_service/models/sembast_scheduled_tx.dart';
-import 'package:provenance_wallet/services/tx_queue_service/models/sembast_tx_signer.dart';
-import 'package:provenance_wallet/services/tx_queue_service/tx_queue_service.dart';
-import 'package:provenance_wallet/services/tx_queue_service/tx_queue_service_error.dart';
+import 'package:provenance_wallet/services/tx_queue_client/models/sembast_gas_estimate.dart';
+import 'package:provenance_wallet/services/tx_queue_client/models/sembast_scheduled_tx.dart';
+import 'package:provenance_wallet/services/tx_queue_client/models/sembast_tx_signer.dart';
+import 'package:provenance_wallet/services/tx_queue_client/tx_queue_client.dart';
+import 'package:provenance_wallet/services/tx_queue_client/tx_queue_service_error.dart';
 import 'package:provenance_wallet/util/public_key_util.dart';
 import 'package:sembast/sembast.dart';
 import 'package:sembast/sembast_memory.dart';
 
-class DefaultQueueTxService implements TxQueueService {
-  DefaultQueueTxService({
+class DefaultQueueTxClient implements TxQueueClient {
```

Comment:
I think this one should remain with the `Service` suffix since it doesn't wrap a web service.